meson.build

```
@@ -212,20 +202,26 @@ if enable_logind
 endif
 config_data.set('HAVE_LIBSYSTEMD', enable_logind)
 
-systemd_dep = dependency('systemd').found()
-if systemd_dep
-  tmpfiles_conf = files(
-    'data/polkit-tmpfiles.conf'
-  )
+config_data.set('HAVE_PIDFD_OPEN', cc.get_define('SYS_pidfd_open', prefix: '#include <sys/syscall.h>') != '')
 
-  install_data(
-    tmpfiles_conf,
-    install_dir: pk_prefix / 'lib/tmpfiles.d/'
-  )
+# systemd unit / sysuser / tmpfiles.d file installation directories
+systemdsystemunitdir = get_option('systemdsystemunitdir')
+systemd_dep = dependency('systemd')
```

```suggestion
systemd_dep = dependency('systemd', required : false)
```
this is necessary to build without systemd

otherwise, this works well and successfully builds and installs in void's package build system with `-Dintrospection=true  -Dman=true  -Dsession_tracking=elogind -Dpam_prefix=/usr/lib/pam.d`